src/components/ReceiptAudit.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import {View} from 'react-native';
+import useLocalize from '@hooks/useLocalize';
+import useTheme from '@hooks/useTheme';
+import useThemeStyles from '@hooks/useThemeStyles';
+import Icon from './Icon';
+import * as Expensicons from './Icon/Expensicons';
+import Text from './Text';
+
+function ReceiptAuditHeader({notes = [], showAuditMessage = false}: {notes?: string[]; showAuditMessage?: boolean}) {
+    const styles = useThemeStyles();
+    const theme = useTheme();
+    const {translate} = useLocalize();
+
+    const issuesFoundText = notes.length > 0 ? translate('iou.receiptIssuesFound', notes.length) : translate('common.verified');
+    return (
+        <View style={[styles.ph5, styles.mbn1]}>
+            <View style={[styles.flexRow, styles.alignItemsCenter]}>
+                <Text style={[styles.textLabelSupporting]}>{translate('common.receipt')}</Text>
+                {showAuditMessage && (
+                    <>
+                        <Text style={[styles.textLabelSupporting]}>{` • ${issuesFoundText}`}</Text>
```

Comment:
Something is off here, so let's test the Spanish version too. In English this says "Issue(s) Found", and in Spanish this will say "Problema(s)", but it should say "Problema(s) encontrado(s)" to match English and to make sense

Reply:
Needs to be
```
receiptIssuesFound: (count: number) => `${count === 1 ? 'Problema encontrado' : 'Problemas encontrados'} `,
```